README.md

```diff
@@ -232,6 +232,42 @@ GhostInspector.downloadSuiteSeleniumJson('[suite-id]', 'suite.zip', function (er
 });
 ```
 
+#### GhostInspector.downloadSuiteGhostJson(suiteId, dest, [callback])
+Download a file of all tests in this suite in Ghostinspector JSON format
+
+```js
+// Example using await
+try {
+    await GhostInspector.downloadSuiteGhostJson('[suite-id]', 'suite.json');
```

Comment:
@justinklemm what do you think of this naming? I think my preference would be to just go with `downloadSuiteJson(...)` for simplicity, any thoughts?

Reply:
Yea, I think `downloadSuiteJson` probably makes the most sense. For other export formats that happen to be JSON, we can use a more specific method name.